docs/pages/printing_text.rst

```diff
@@ -272,3 +273,50 @@ applied as well.
     text = to_formatted_text(html, style='class:my_html bg:#00ff00 italic')
 
     print_formatted_text(text)
+
+
+String concatenation
+^^^^^^^^^^^^^^^^^^^^
+
+Formatted text can be concatenation (or merged) together, with the
```

Comment:
Several typos:

- "text can be **concatenation** (or merged)"

- "take multiple text object **with** either are plain text"

- "will **concatente** them into"